tests/orderly/test_run.py

```diff
@@ -0,0 +1,79 @@
+import shutil
+
+import pytest
+from orderly.run import _validate_src_directory, orderly_run
+
+from outpack.init import outpack_init
+from outpack.root import root_open
+
+
+## We're going to need a small test helper module here at some point,
+## unfortunately pytest makes that totally unobvious how we do it, but
+## we'll get there. For now inline the code as we use it.
```

Comment:
I think possibly [fixtures](https://docs.pytest.org/en/6.2.x/fixture.html) might be the pythonic way to it